Objective-C/Tests/LogTest.m

```diff
@@ -57,33 +49,24 @@ - (void) tearDown {
     [self restoreLoggerConfig];
 }
 
-- (CBLLogFileConfiguration*) logFileConfig {
-    return [[CBLLogFileConfiguration alloc] initWithDirectory: logFileDirectory];
-}
-
 - (void) backupLoggerConfig {
-    _backup = [[FileLoggerBackup alloc] init];
-    _backup.level = CBLDatabase.log.file.level;
-    _backup.config = CBLDatabase.log.file.config;
-    _backupConsoleLevel = CBLDatabase.log.console.level;
-    _backupConsoleDomain = CBLDatabase.log.console.domains;
+    _backup = [[CBLFileLogSink alloc] initWithLevel: CBLLogSinks.file.level directory: logFileDirectory];
+    _backupConsoleLevel = CBLLogSinks.console.level;
+    _backupConsoleDomain = CBLLogSinks.console.domain;
```

Comment:
The backup could just be some thing like this:
```
_file = CBLLogSinks.file;
_console = CBLLogSinks.console;
```

I think after all the only backup is the console log sink for the case that we have enabled the console log sink for debugging purpose.